test-utils/src/main/java/io/ballerina/stdlib/log/testutils/utils/OSUtils.java

```diff
@@ -78,4 +79,14 @@ public static Process processFromObject(BObject objVal) {
     public static boolean isValidDateTime(BString dateTime) {
         return dateTime.substring(0, 23).getValue().matches("^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}.\\d{3}");
     }
+
+    /**
+     * Get the environment variable value.
+     * @param key Environment variable key
+     * @return  Environment variable value
+     */
+    public static BString getEnv(BString key) {
+        String envValue = System.getenv(key.getValue());
+        return StringUtils.fromString(Objects.requireNonNullElse(envValue, ""));
```

Comment:
Can you use the `os:getEnv`[1] function instead

1. https://central.ballerina.io/ballerina/os/1.8.0#getEnv

Reply:
> Can you use the `os:getEnv`[1] function instead
> 
> 1. https://central.ballerina.io/ballerina/os/1.8.0#getEnv

I tried to use this, but it says that the "os" module is not found. I assume the workaround is to include the native jar of "os" module in the libs and add it as a platform dependency just like the "io" module. What approach is better?